velox/exec/Task.cpp

```diff
@@ -927,6 +927,7 @@ void Task::resume(std::shared_ptr<Task> self) {
         if (driver == nullptr) {
           continue;
         }
+        self->driversClosedByTask_.emplace_back(driver);
```

Comment:
Let's move to the place where we put into offThreadDrivers? Those are closed by task and moved out of drivers_ slot.

Reply:
@xiaoxmeng
I don't think it is a good idea. There are some conditions that prevent drivers from being dded to offThreadDrivers.
I believe it is important to add such drivers too.

Reply:
Those drivers will stay in drivers_ so we can see those drivers if they have problem.